src/components/cart/CartItem/CartItem.tsx

```diff
@@ -20,7 +21,7 @@ function CartItem({ item }: Props) {
   const { updateItemQuantity } = useCart()
 
   return (
-    <article
+    <Card
```

Comment:
Why didn't you use the other card elements (`CardContent`, `CardImage`, `CardActions` ...) to compound this `CartItem`?

Reply:
Didn't know about them. Will look at it. Thanks

Reply:
okay! I just asked, but you don't have to use it if not seems necessary :)